cmd/dao_tests/pubkey_test.go

```diff
@@ -0,0 +1,66 @@
+//go:build integration
+// +build integration
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/RHEnVision/provisioning-backend/internal/dao"
+	_ "github.com/RHEnVision/provisioning-backend/internal/dao/sqlx"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/db"
+)
+
+func Setup(t *testing.T, s string) (dao.PubkeyDao, context.Context, error) {
+	err := db.Seed("dao_test")
+	if err != nil {
+		assert.Nil(t, err, fmt.Sprintf("Error purging the database: %v", err))
+		return nil, nil, err
+	}
+	ctx := context.Background()
+	pkDao, err := dao.GetPubkeyDao(ctx)
+	if err != nil {
+		assert.Nil(t, err, fmt.Sprintf("%s test had failed: %v", s, err))
+		return nil, nil, err
+	}
+	return pkDao, ctx, nil
+}
+
+func TearDown() {
+	config.Initialize()
+
+	err := db.Initialize()
+	if err != nil {
+		fmt.Errorf("Error initializing database: %v", err)
```

Comment:
fmt.Errorf only creates the error, but does not do anything with it, should we do t.Errorf instead?
Or if we move this to TestMain, then we can keep this, but panic(fmt....)